install_phantomjs2_debian.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# This script install PhantomJS in your Debian/Ubuntu System
+# https://gist.github.com/julionc/7476620
+#
+# This script must be run as root:
+# sudo sh install_phantomjs.sh
+#
+
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root" 1>&2
+    exit 1
+fi
+
+PHANTOM_VERSION="phantomjs-2.1.1"
```

Comment:
Why not just use the `latest` tarball, instead of having to revise this script every time a new version is out?